bucket/libreoffice.json

```diff
@@ -1,62 +1,69 @@
+            "hash": "509c70c1c8136805480146b55e4bad5dc73b11ee47b4682b43cf07670109e176"
+        },
+        "32bit": {
+            "url": "https://download.documentfoundation.org/libreoffice/stable/7.3.4/win/x86/LibreOffice_7.3.4_Win_x86.msi",
+            "hash": "e73e2fd3d95f305a8517abe65c940e2b7de13d4271fb789f38f89c28df57933a"
+        }
     },
-    "url": "https://download.documentfoundation.org/libreoffice/portable/7.3.2/LibreOfficePortable_7.3.2_MultilingualStandard.paf.exe#/dl.7z",
-    "hash": "ae81da2a2bfb27a0cbdeffa65f712f8483a4f20049cbd1e02de2a36e6c3ff9d5",
     "pre_install": [
-        "if (!(Test-Path \"$persist_dir\\Data\\settings\")) {",
-        "    New-Item \"$dir\\Data\\settings\\LibreOfficePortableSettings.ini\" -Value \"[LibreOfficePortableSettings]`nInvalidPackageWarningShown=$version.0\" -Force | Out-Null",
-        "    if (Test-Path \"$env:AppData\\LibreOffice\") {",
-        "        info \"Copying old '$env:AppData\\LibreOffice' to '$persist_dir\\Data\\settings'\"",
-        "        Get-Item \"$env:AppData\\LibreOffice\\*\\*\" | Copy-Item -Destination \"$dir\\Data\\settings\" -Recurse -Force",
-        "    } else {",
-        "        Copy-Item \"$dir\\App\\DefaultData\\*\" \"$dir\\Data\" -Recurse -Force",
-        "    }",
-        "}",
-        "Remove-Item \"$dir\\`$*\" -Recurse -Force"
+        "Remove-Item \"$dir\\`$*\" -Recurse -Force",
+        "New-Item \"$dir\\LibreOffice\\share\\fonts\" -ItemType Directory | Out-Null",
+        "Move-Item \"$dir\\Fonts\" \"$dir\\LibreOffice\\share\\fonts\\truetype\" -Force"
     ],
     "shortcuts": [
         [
-            "LibreOfficePortable.exe",
+            "LibreOffice\\program\\soffice.exe",
             "LibreOffice\\LibreOffice"
         ],
         [
-            "LibreOfficeBasePortable.exe",
+            "LibreOffice\\program\\sbase.exe",
             "LibreOffice\\LibreOffice Base"
         ],
         [
-            "LibreOfficeCalcPortable.exe",
+            "LibreOffice\\program\\scalc.exe",
             "LibreOffice\\LibreOffice Calc"
         ],
         [
-            "LibreOfficeDrawPortable.exe",
+            "LibreOffice\\program\\sdraw.exe",
             "LibreOffice\\LibreOffice Draw"
         ],
         [
-            "LibreOfficeImpressPortable.exe",
+            "LibreOffice\\program\\simpress.exe",
             "LibreOffice\\LibreOffice Impress"
         ],
         [
-            "LibreOfficeMathPortable.exe",
+            "LibreOffice\\program\\smath.exe",
             "LibreOffice\\LibreOffice Math"
         ],
         [
-            "LibreOfficeWriterPortable.exe",
+            "LibreOffice\\program\\swriter.exe",
             "LibreOffice\\LibreOffice Writer"
         ]
     ],
-    "persist": "Data",
     "checkver": {
-        "url": "https://download.documentfoundation.org/libreoffice/portable/?C=M;O=D",
+        "url": "https://download.documentfoundation.org/libreoffice/stable/?C=M;O=D",
```

Comment:
`libreoffice-fresh`'s checkver updates to 7.3.4.2, which was released 3rd June 2022. Should we copy that checkver too?

[Chocolatey](https://community.chocolatey.org/packages/libreoffice-fresh) and [Homebrew](https://formulae.brew.sh/cask/libreoffice) stick to 7.3.4 but [WinGet](https://github.com/microsoft/winget-pkgs/tree/master/manifests/t/TheDocumentFoundation/LibreOffice/7.3.4.2) uses 7.3.4.2.

Checking for updates within LibreOffice 7.3.4 reports that "LibreOffice 7.3 is up to date."

Reply:
> libreoffice-fresh's checkver updates to 7.3.4.2, which was released 3rd June 2022. Should we copy that checkver too?
Of course. We should track any minor update if possible, so that hash error / installtion error will not happen.

Reply:
Done. I modified the checkver based on `libreoffice-fresh`'s checkver. It uses `.../latest/win/x86_64/...` to get the version number, hopefully that will be okay.